lib/wanda/messaging/mapper.ex

```diff
@@ -33,13 +37,52 @@ defmodule Wanda.Messaging.Mapper do
     ExecutionCompleted.new!(
       execution_id: execution_id,
       group_id: group_id,
-      result: to_result(result)
+      result: map_result(result)
     )
   end
 
-  defp to_result(:passing), do: :PASSING
-  defp to_result(:warning), do: :WARNING
-  defp to_result(:critical), do: :CRITICAL
+  @spec from_execution_requested(ExecutionRequested.t()) :: %{
+          execution_id: String.t(),
+          group_id: String.t(),
+          targets: [Target.t()],
+          env: %{String.t() => boolean() | number() | String.t()}
+        }
+  def from_execution_requested(%ExecutionRequested{
+        execution_id: execution_id,
+        group_id: group_id,
+        targets: targets,
+        env: env
+      }) do
+    %{
```

Comment:
I have doubts whether we should create a specific module type instead of returning plain map

Reply:
it's ok to use a map for this I wouldn't go as far as creating a struct for this since it's used internally and it doesn't go in or out our domain